### ISSUE TITLE
Update wording and case of Settings options

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/universalviewer#1]
The Settings modal currently looks like this:

----
![strad___and_other_stories___stanford_digital_repository](https://user-images.githubusercontent.com/101482/28292198-da7c10ba-6b02-11e7-9b0d-f8f0dee33086.png)
----
The settings might be more readable and understandable if we change all options to use sentence-case rather than title-case and update the wording of the options. So the options would read like this:

- Enable page navigator
- Enable two-page view
- Click mouse to zoom
- Preserve zoom level across pages

I'm not 100% sure about the two-page view option because it seems like it might not be working correctly. Is it supposed to disable two-page view if you uncheck it, or just change the default? Because currently if I uncheck it, the two-page view of the object switches to a single-page, but I can still get back to it by clicking the two-page icon. And then when I return to Settings the option is selected again. So it seems like the setting just controls what is the default. (But if so, I'm not sure this justifies even being a setting? I must be missing something.)

@jvine Feel free to update the wording of the options if you have better ideas.

**Comment by edsilv** [comment]
@ggeisler I agree that Two Page View could be removed from settings as it's now just duplicating the function of the icons next to the cog. We may need to run this past the community however as I might be forgetting something about why it's important.

I also prefer sentence case instead of title case. This is a config option and anyone can change this text to whatever they like, but I'd suggest that we make this the default.

Let's post this thread to the UX channel on the UV Slack to gather some opinion.

**Comment by mialondon** [comment]
@ggeisler my old usability notes suggest changing the 'navigator' line to 'Turn on overview box' or something similar, as no-one we talked to knew what a navigator was

**Comment by mialondon** [comment]
And the BL's version also says 'Keep zoom level the same between pages' - wordy but hopefully clear

**Comment by ggeisler** [comment]
@edsilv Sounds good.

@mialondon Sure, I'm not surprised that "navigator" wasn't always understood. Maybe something like "Enable overview box" or "Display overview box" would work ("enable" and "display" seem slightly better than "turn on" to me)?

And 'Keep zoom level the same between pages' is a definite improvement. Something like "'Maintain zoom level between pages' is a bit shorter and might also work.
